main.nf

```diff
@@ -22,14 +23,15 @@ def helpMessage() {
     Usage:
 
     nextflow run FredHutch/cellprofiler-batch-nf <ARGUMENTS>
-    
+
     Required Arguments:
       --input_h5            Batch file created by the CellProfiler GUI interface defining the analysis to run
       --input_txt           List of images to process using the specified analysis
       --output              Path to output directory
 
     Optional Arguments:
       --n                   Number of images to analyze in each batch (default: 1000)
+      --reduce_n            Number of tabular results to combine/concatenate in the first round (default: 100)
```

Comment:
The "reduce" terminology is most clear in the context of the "map-reduce" approach, and may be misleading out of context. When I first read the param name I thought it was going to apply a reduce action to the `--n` flag. Recommend changing to `--concat_n`